src/hotspot/share/gc/shared/stringdedup/stringDedupConfig.cpp

```diff
@@ -116,7 +116,7 @@ size_t StringDedup::Config::desired_table_size(size_t entry_count) {
 bool StringDedup::Config::ergo_initialize() {
   if (!UseStringDeduplication) {
     return true;
-  } else if (!UseG1GC && !UseShenandoahGC && !UseZGC) {
+  } else if (UseEpsilonGC || UseSerialGC) {
```

Comment:
This change is a small land mine for the next new GC project.  I'd prefer adding !UseParallelGC.  Or rewrite to have an `else if (UseXXXGC || ...) { ... } else { // not supported }` structure.